docs/Installing.asciidoc

```diff
@@ -25,9 +25,13 @@ installation, useful for a single user setup. Else, continue with the more
 advanced section about "`Custom installation - Repositories and procedure''.
 
 
-== openQA quick bootstrap
+== Quick bootstrapping under openSUSE
 
-To quickly get a working openQA installation, you can use openQA-bootstrap.
+To quickly get a working openQA installation, you can use the openqa-bootstrap script.
+It esentially automates the steps mentioned in the
+<<#custom_installation,Custom installation>> section. Under Leap it will add repositories
+from the OBS project +devel:openQA+ and under Tumbleweed install the packages from the
+official repository.
```

Comment:
IMHO this is duplicating too much and too much details for the quick bootstrap which is really intended for people that don't care about doing stuff manually. To keep it simple I strongly suggest to keep this section very terse

Reply:
Ok, I'm removing the last sentence. But I find it nice to have at least one statement what this script actually does.